docs/TEST-GUIDELINES.md

```diff
@@ -0,0 +1,29 @@
+# Test guidelines
+
+Automated tests are very important for building software. They let us add, modify and understand the code in our app with confidence and can prevent us from making mistakes that could end up effecting people collecting data in the field.
+
+Almost any Pull Request for Collect will require a set of tests along with its code changes. Otherwise there will be nothing to ensure that that code continues to work or exist in the future. By far easiest way to make sure you have tests for the changes you're making is to follow a [TDD](https://en.wikipedia.org/wiki/Test-driven_development) style of development and write a test before each change/addition to the code.
```

Comment:
"By far easiest way to make sure you have tests for the changes you're making is to follow" -> "We recommend following"